website/docs/d/notification_destination.html.markdown

```diff
@@ -33,18 +33,44 @@ resource "newrelic_notification_channel" "foo-channel" {
 }
 ```
 
+## Name Example Usage
+
+```hcl
+# Data source
+data "newrelic_notification_destination" "foo" {
+  name = "webhook-destination"
+}
+
+# Resource
+resource "newrelic_notification_channel" "foo-channel" {
+  name = "webhook-example"
+  type = "WEBHOOK"
+  destination_id = data.newrelic_notification_destination.foo.id
+  product = "IINT"
+
+  property {
+    key = "payload"
+    value = "{\n\t\"name\": \"foo\"\n}"
+    label = "Payload Template"
+  }
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
+Required **one of these two** arguments:
 * `id` - (Required) The id of the notification destination in New Relic.
+* `name` - (Required) The name of the notification destination.
```

Comment:
Just a thought - I think we can have (Optional) against **id** and **name** here, with the note at the top being "Either of the following two attributes are required, and not both:"